### ISSUE TITLE
Default paginationSize documentation is incorrect

### ISSUE THREAD

**Issue by iursevla** [olifolkerd/tabulator#2914]
**Website Page**
http://tabulator.info/docs/4.7/page#local

The following is writtent:

```js
var table = new Tabulator("#example-table", {
    pagination:"local", //enable local pagination.
    paginationSize:5, // this option can take any positive integer value (default = 10)
});
```

**Describe the issue**
The default isn't 10. If you don't pass a paginationSize value it will be calculated like:

https://github.com/olifolkerd/tabulator/blob/673498817be34f9e229901de950959aedf819019/src/js/modules/page.js#L215

---

#2656

**Comment by olifolkerd** [comment]
Hey  @iursevla 

Thanks for letting me know, i will make sure the documentation gets updated.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
I have made the fix my end, i will push it live with the 4.8 site updates later this month.

Cheers

Oli :)
